### PR TITLE
[installer]: bump rabbitmq chart to remove deprecated kube api

### LINKED ISSUE #6829: [Installer]: update RabbitMQ Helm chart when PodDisruptionBudget update applied
We're using `8.16.0` of the Bitnami RabbitMQ chart. This gives this warning:

```
Warning: policy/v1beta1 PodDisruptionBudget is deprecated in v1.21+, unavailable in v1.25+; use policy/v1 PodDisruptionBudget
```

A [PR](https://github.com/bitnami/charts/pull/8018) has been created to apply the suggestion in the warning. When this is merged and released, we should upgrade our RabbitMQ version to apply this change

### PR DESCRIPTION
**Pull request by MrSimonEmms**
## Description

RabbitMQ used the deprecated pod disruption budget API. This has been upgraded in the latest version of the chart.

As we use `helm template` to generate the YAML, this [fakes the Kubernetes version](https://github.com/helm/helm/issues/7991). There, we have to set the `kubeVersion` global in the RabbitMQ template

## Related Issue(s)

Fixes #6829

## How to test

Deploy an instance with the Installer.

## Release Notes

```release-note
[Installer]: Update RabbitMQ chart
```

### CODE REVIEW

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please ask for approval from **mrsimonemms** after the PR has been reviewed.

Associated issue: *#6829*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

<details open>
Needs approval from an approver in each of these files:

- **[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Review by princerachit — APPROVED** [review]

**Comment by roboquat** [comment]
LGTM label has been added.  <details>Git tree hash: 1982fa9ef5ec7ccdd912b1c6448ec262c4d486fb</details>

**Comment by roboquat** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/gitpod-io/gitpod/pull/7041#pullrequestreview-827377356" title="Approved">princerachit</a>*

Associated issue: *#6829*

The full list of commands accepted by this bot can be found [here](https://prow.gitpod-dev.com/command-help?repo=gitpod-io%2Fgitpod).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[installer/OWNERS](https://github.com/gitpod-io/gitpod/blob/main/installer/OWNERS)~~ [princerachit]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Pull request merged**
